### ISSUE TITLE
octavemagic and addpath

### ISSUE THREAD

**Issue by cancan101** [blink1073/oct2py#65]
How do I use the `octavemagic` in IPython Notebook along with `addpath`?

This works:
```M
from oct2py import octave
octave.addpath('/home/ubuntu/Downloads/')
octave.monofilt(a, 3, 4, 2, 0.65)
```

but then this:
```
%octave monofilt(a) -i a
```
leads to:
```
---------------------------------------------------------------------------
OctaveMagicError                          Traceback (most recent call last)
<ipython-input-12-2814f62667cf> in <module>()
----> 1 get_ipython().magic(u'octave monofilt(a) -i a')

/usr/local/lib/python2.7/dist-packages/IPython/core/interactiveshell.pyc in magic(self, arg_s)
   2203         magic_name, _, magic_arg_s = arg_s.partition(' ')
   2204         magic_name = magic_name.lstrip(prefilter.ESC_MAGIC)
-> 2205         return self.run_line_magic(magic_name, magic_arg_s)
   2206 
   2207     #-------------------------------------------------------------------------

/usr/local/lib/python2.7/dist-packages/IPython/core/interactiveshell.pyc in run_line_magic(self, magic_name, line)
   2124                 kwargs['local_ns'] = sys._getframe(stack_depth).f_locals
   2125             with self.builtin_trap:
-> 2126                 result = fn(*args,**kwargs)
   2127             return result
   2128 

/usr/local/lib/python2.7/dist-packages/oct2py/ipython/octavemagic.pyc in octave(self, line, cell, local_ns)

/usr/local/lib/python2.7/dist-packages/IPython/core/magic.pyc in <lambda>(f, *a, **k)
    191     # but it's overkill for just that one bit of state.
    192     def magic_deco(arg):
--> 193         call = lambda f, *a, **k: f(*a, **k)
    194 
    195         if callable(arg):

/usr/local/lib/python2.7/dist-packages/oct2py/ipython/octavemagic.pyc in octave(self, line, cell, local_ns)
    312             msg = re.sub('"""\s+', '"""\n', msg)
    313             msg = re.sub('\s+"""', '\n"""', msg)
--> 314             raise OctaveMagicError(msg)
    315 
    316         key = 'OctaveMagic.Octave'

OctaveMagicError: Oct2Py tried to run:
"""
monofilt(a)
"""
Octave returned:
'monofilt' undefined near line 3 column 13
```

**Comment by blink1073** [comment]
@cancan101, `from oct2py import octave` gives you an *instance* of `Oct2Py`, which is not the same as the one used by the octavemagic, so you'd have to use `%octave addpath('/home/ubuntu/Downloads/')` to get your desired result.

**Comment by blink1073** [comment]
I switched it to use `octave` by default in 7d560101a06bb9d379b48e26faca90407ff22724, so it will be less confusing in the next release.

**Comment by cancan101** [comment]
I just tested the dev code out and it works great.

**Comment by blink1073** [comment]
Fixed in 3.0.
